static/app/views/projectsDashboard/index.tsx

```diff
@@ -21,40 +20,40 @@ import ProjectsStatsStore from 'app/stores/projectsStatsStore';
 import space from 'app/styles/space';
 import {Organization, TeamWithProjects} from 'app/types';
 import {sortProjects} from 'app/utils';
+import useTeams from 'app/utils/useTeams';
 import withApi from 'app/utils/withApi';
 import withOrganization from 'app/utils/withOrganization';
-import withTeamsForUser from 'app/utils/withTeamsForUser';
 
 import Resources from './resources';
 import TeamSection from './teamSection';
 
 type Props = {
   api: Client;
   organization: Organization;
-  teams: TeamWithProjects[];
```

Comment:
This `TeamsWithProjects` type seems questionable 🤔

Reply:
can we use another type here or maybe update this one?

Reply:
yea this is a little unfortunate. I should update the regular team type since I think when we fetch teams we always get a list of associated projects.